opm/autodiff/BlackoilPropsAdFromDeck.cpp

```diff
@@ -166,116 +168,35 @@ BlackoilPropsAdFromDeck::BlackoilPropsAdFromDeck(const BlackoilPropsAdFromDeck&
 
         phase_usage_ = phaseUsageFromDeck(deck);
 
+        gasPvt_ = std::make_shared<GasPvt>();
+        oilPvt_ = std::make_shared<OilPvt>();
+        waterPvt_ = std::make_shared<WaterPvt>();
 
+        gasPvt_->initFromDeck(deck, eclState);
```

Comment:
Unconditionally initializing PVT for the pahse causes exception when the PVT for the phase is not specified  in the deck.